guide/sections/part1/data-publisher.adoc

```diff
@@ -1,23 +1,204 @@
+
+The URL included in a notification message that is used to access Core data from a WIS2 Node, or the "canonical" URL if multiple URLs are provided, must:
+
+. Refer to an individual data object; and
+. Be directly resolvable, i.e., the data object can downloaded simply by resolving the given URL without further action.
+
+A Global Cache will download and cache the data object accessed via this URL.  
+
+The Global Caches are designed to support Members efficiently share real-time and near real-time data; they take on the task of making sure that Core data is available to all and cover the costs of delivering data to a global community.
+
+Unfortunately, Global Caches cannot republish _all_ Core data: there is a limit to how much data they can afford to serve. Currently, a Global Cache expected to cache about 100GB of data each day.
+
+If frequent updates to your dataset are very large (e.g., weather prediction models or remote sensing observations) you will need to share the burden of distributing your data with the Global Cache operators. You should work with your GISC to determine the highest priority elements of your Core datasets that will be republished by the Global Caches.
+
+For Core data that is not to be cached, you must set the ``cache`` property in the notification message to ``false``footnote:[Default value for the ``cache`` property is ``true``; omission of the property will result in the data object being cached.]. 
+
+You must ensure that Core data that is not cached is publicly accessible from your WIS2 Node; i.e., with no access control mechanisms in place.
+
+A Global Cache operator may choose to disregard your cache preference - for example, if they feel that the content you are providing is large enough to impede provision of caching services for other Membersfootnote:[Excessive data volume isn't the only reason they may refuse to cache content. Other reasons include: too many small files, unreliable download from a WIS2 Node, etc.]. In such cases, the Global Cache operator will log this behaviour. In collaboration with the Global Cache operators, your GISC will work with you to resolve concerns. 
+
+Finally, please note that Global Caches are under no obligation to cache data published on _experimental_ topics. For such data, the ``cache`` property should be set to ``false``.
+
+===== Implementing access control
+
+Recommended data, as defined in the WMO Unified Data Policy (Resolution 1 (Cg-Ext 2021)), is exchanged on WIS2 in support of Earth system monitoring and prediction efforts and _may_ by provided with conditions on use. This means that you may control access to Recommended data.
+
+Access control should use only the "security schemes" for authentication and authorization specified in OpenAPIfootnote:[OpenAPI security schemes https://swagger.io/docs/specification/authentication/].
```

Comment:
Suggest using https://spec.openapis.org/oas/v3.1.0#security-scheme-object as reference

Reply:
Fixed. Will also clean up references to this in part2/operations.adoc and part2/wis2node.adoc